platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerView.swift

```diff
@@ -62,13 +62,6 @@ public struct WysiwygComposerView: UIViewRepresentable {
     }
 
     public func updateUIView(_ uiView: PlaceholdableTextView, context: Context) {
-        Logger.textView.logDebug(
-            [
-                viewModel.attributedContent.logSelection,
-                viewModel.attributedContent.logText,
-            ],
-            functionName: #function
-        )
```

Comment:
Weird that this log slows down things. I would expect the logs containing `viewModel.content` to slow things down, but not the one containing `viewModel.attributedContent` 😅

Reply:
I don't think is the log itself slowing it down, in the end this is called everytime the view is updated, I just tested on element-ios that if a state value is changed frequently internally in swift ui, for example to create a gesture animation, the animation, the view regenerate through the updateUIView function, which is called multiple times.
I just feel that the log slows everything down when the function is called multiple times, since it makes the animations look like they are stuttering.